wg-async.toml

```diff
@@ -1,28 +1,58 @@
 name = "Rust Async Working Group Team"
 description = "Meetings for the async working group"
 
+# Active recurring events
+
 [[events]]
-uid = "1704471961833"
-title = "wg-async Weekly Meeting"
-description = "First Thursday of each month is sprint planning. On other weeks we do deep dives, reading clubs, and open discussion."
-location = "#wg-async on Zulip (https://rust-lang.zulipchat.com/#narrow/stream/187312-wg-async)"
-last_modified_on = "2024-01-11T10:00:00.00Z"
-start = "2024-01-11T17:00:00.00Z"
-end = "2024-01-11T18:00:00.00Z"
+uid = "1626e53f9596461fcb0ec39d032178cd0afe5943"
+title = "WG-async Design Meeting"
+description = "https://github.com/orgs/rust-lang/projects/40"
```

Comment:
Just a minor suggestion, it wouldn't hurt to have a one sentence description of the meeting along with the link for more details, both here and on the other events.

Reply:
(Just to be clear, I don't see this as a blocker; feel free to update if you want or merge as-is if you prefer.)

Reply:
Thanks, I've added descriptions.